src/opsinputs/opsinputs_varobswriter_mod.F90

```diff
@@ -963,6 +963,9 @@ subroutine opsinputs_varobswriter_populateobservations( &
       ! TODO(someone): handle this varfield
       ! call Ops_Alloc(Ob % Header % AMSUb_Temp, "AMSUb_Temp", Ob % Header % NumObsLocal, Ob % AMSUb_Temp)
     case (VarField_cloud)
+      call opsinputs_fill_fillelementtype2dfromnormalvariablewithlevels( &
+        Ob % Header % Cloud, "Cloud", Ob % Header % NumObsLocal, Ob % Cloud, &
```

Comment:
Not necessarily, but maybe.

The way it works in OPS is that `Cloud` or `cloud` is the field that goes into the varobs, i.e. this one, it is the cloud fraction on model levels calculated in the SurfaceCloud routine, whereas the CX field is what is called `level_cloud` and I think is the one Fabien has added there. I'm not 100% sure what their interaction should be in JOPA, whether they should have the same name or not.

Reply:
I think this needs to match with the name proposed in https://github.com/JCSDA-internal/ioda/pull/1062, i.e. `cloudAmount`  instead of `Cloud`.

However I am happy if you want to commit this as is, and then handle the variable name change with separate PR to ioda, ufo, opsinputs, etc?